syntax = "proto3";

package transaction;

message Transaction {
  optional string t_time = 1;
  optional int64 t_id = 2;
  optional int64 t_customer_id = 3;
  optional double t_amount = 4;
} 